src/HealthGPS.Core/datatable.cpp

```diff
@@ -46,15 +46,13 @@ const DataTableColumn &DataTable::column(const std::string &name) const {
     throw std::out_of_range(fmt::format("Column name: {} not found.", name));
 }
 
-const std::pair<bool, DataTableColumn &>
+const std::optional<std::reference_wrapper<const DataTableColumn>>
```

Comment:
The `std::optional` bit doesn't need to be `const`. You don't want the user to muck about with the `DataTableColumn` inside, but they can reassign the returned value etc.

```suggestion
std::optional<std::reference_wrapper<const DataTableColumn>>
```